pkg/logging/logger.go

```diff
@@ -26,7 +26,7 @@ var logger *zap.SugaredLogger
 type loggerKey struct{}
 
 func init() {
-	zapLogger, _ := zap.NewProduction()
+	zapLogger, _ := zap.NewDevelopment(zap.AddCaller(), zap.AddStacktrace(zap.ErrorLevel))
```

Comment:
The Development setting is required to enable the call stack to be displayed across multiple lines.

I will refactor the code to remove the init function and create a separate PR to configure the zap logger to either development or production based on configuration.

Reply:
I have updated the PR to utilize an ENV variable to facilitate loading the production zap logger or the development one.